### PR TITLE
chore: fix V3 js templates and action

### PR DESCRIPTION
**Pull request by blackchoey**
1. botAadApp/create should output `SECRET_BOT_PASSWORD`
2. update sso-tab-js deploy folder
3. update notification-restify-js launch.json

### CODE REVIEW

File: templates/scenarios/notification-restify-js/.appserviceIgnore [hunk 1]
```diff
@@ -12,6 +12,7 @@ local.settings.json
 test
 tsconfig.json
 .DS_Store
+node_modules
```
Comment (Siglud) [inline templates/scenarios/notification-restify-js/.appserviceIgnore:15]: We need to keep the node modules folder in package, cause RUN_FROM_PACKAGE needs all dependencies

File: templates/scenarios/sso-tab-js/teamsfx/app.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ deploy:
   - uses: azureStorage/deploy # Deploy bits to Azure Storage Static Website
     with:
       workingDirectory: .
```
Comment (Siglud) [inline templates/scenarios/sso-tab-js/teamsfx/app.yml:33]: The working dir can be removed if working in the project path.

File: templates/scenarios/sso-tab-js/teamsfx/app.yml [hunk 3]
```diff
@@ -31,7 +31,7 @@ deploy:
...
-      distributionPath: . # Deploy base folder
+      distributionPath: ./build # Deploy base folder
```
Comment (Siglud) [inline templates/scenarios/sso-tab-js/teamsfx/app.yml:34]: We still need to deploy all the project files, like packages.json, to run the project

Comment (blackchoey) [reply templates/scenarios/sso-tab-js/teamsfx/app.yml:34]: This seems to work, if I use `.`, the index.html is not found, do we need to change the tab url to `https://domain/build/index.html`?

File: templates/scenarios/notification-restify-js/.appserviceIgnore (hunk 1 above)
Comment (blackchoey) [reply templates/scenarios/notification-restify-js/.appserviceIgnore]: thanks, resolved

File: templates/scenarios/sso-tab-js/teamsfx/app.yml (hunk 2 above)
Comment (blackchoey) [reply templates/scenarios/sso-tab-js/teamsfx/app.yml:33]: thanks, resolved

File: templates/scenarios/sso-tab-js/teamsfx/app.yml (hunk 3 above)
Comment (hund030) [reply templates/scenarios/sso-tab-js/teamsfx/app.yml:34]: I think `./build` is correct for tab.

**Review by hund030 — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**